emacs/radian.el

```diff
@@ -43,12 +43,40 @@
 
 ;;; Define utility functions and variables
 
+
+(defcustom radian-org-enable-contrib nil
+  "Non-nil means to make Org contrib modules available.
+This has to be set at the beginning of init, i.e. in the top
+level of init.local.el."
+  :type 'boolean)
+
+(defcustom radian-color-theme-enable t
+  "Non-nil means to load the default Radian color theme.
+Set this to nil if you wish to load a different color theme in
+your local configuration."
+  :type 'boolean)
```

Comment:
These options are no longer necessary since including the relevant packages in `radian-disabled-packages` achieves the same purpose (and is how I implemented them), but I guess they should be kept for backward compatibility.

Reply:
Yeah, that seems reasonable. Can we make it so that Radian reads these variables, and automatically updates `radian-disabled-packages` if they're set to a nonstandard value? We can also mark the original variables as obsolete.

Reply:
Done.